benchmark/framework/containerd_utils.go

```diff
@@ -289,8 +288,19 @@ func GetRemoteOpts(ctx context.Context, platform string) []containerd.RemoteOpt
 	return opts
 }
 
-func GetTestContext() (context.Context, context.CancelFunc) {
-	ctx, cancel := context.WithCancel(context.Background())
+func GetTestContext(logFile io.Writer) (context.Context, context.CancelFunc) {
+	logrus.SetLevel(logrus.InfoLevel)
+	logrus.SetFormatter(&logrus.JSONFormatter{
+		TimestampFormat: log.RFC3339NanoFixed,
+	})
+	if logFile != nil {
+		logrus.SetOutput(logFile)
+	} else {
+		logrus.SetOutput(os.Stderr)
```

Comment:
if no logfile is provided (passing in `nil`) log statements will write to stderr

Reply:
will fix forward